src/components/wizard/HorizontalWizardNav.jsx

```diff
@@ -18,24 +19,31 @@ const HorizontalWizardNav = ({ steps, onNavigate, currentStep }) => {
         onSelect={(key) => onNavigate(parseInt(key))}
       >
         {steps.map((step, index) => (
-          <NavItem key={"nav" + index} id={"wizard-nav-" + index}>
-            <NavLink
-              eventKey={index}
-              active={index === currentStep ? "active" : ""}
-              hidden={options.debugMode ? false : !FormUtils.isRelevant(step)}
-              className={classNames([
-                options.debugMode && !FormUtils.isRelevant(step)
-                  ? "show-irrelevant"
-                  : Question.getEmphasizedClass(step),
-                "wizard-nav",
-              ])}
-            >
-              {JsonLdUtils.getLocalized(
-                step[JsonLdUtils.RDFS_LABEL],
-                options.intl
-              )}
-            </NavLink>
-          </NavItem>
+          <IconOverlay
+            id="step-disabled"
+            tooltipContent="Navigation to this tab is disabled to prevent overlooking important questions. Use form-related buttons to navigate to this tab instead."
```

Comment:
Don't we have i18n ?

Reply:
Not really, unfortunately... We have an old ticket for that: https://github.com/kbss-cvut/s-forms/issues/193. I believe this should be re-worked in the same way it is done in record-manager-ui, as for now i18n is in a very bad state in s-forms.

Reply:
![image](https://github.com/kbss-cvut/s-forms/assets/3962237/d233fbd5-e806-4efa-abda-f483364ebcd0)